server/config/service.go

```diff
@@ -112,6 +112,9 @@ func (s *service) Reconfigure(stored StoredConfig, services ...Configurable) err
 }
 
 func (s *service) StoreConfig(sc StoredConfig) error {
+	if strings.HasPrefix(*s.mattermostConfig.ServiceSettings.SiteURL, "http://localhost:") {
```

Comment:
This is done because `s.mm.Configuration.SavePluginConfig` method is not working on an e2e environment.